docs/github-general.md

```diff
@@ -62,6 +62,59 @@ Add this new SSH key to your profile in GitHub by following [these instructions]
     This will download all project data from the associated Amazon S3 bucket and create symlinks in `data/` to the latest data release version. This command can be re-run to ensure that the most updated files are downloaded. 
     
 
+## Data release instructions
+
+For generating a new versioned data release, follow the steps below:
+
+1. **Create a versioned folder in S3**
+
+    Create the new version folder in the designated S3 bucket:
+
+        s3://<bucket-name>/<repo-name>/v1/
+
+    Upload all finalized data files for this release into the versioned folder.
+
+2. **Generate MD5 checksums for uploaded files**
+
+    Run the following command from your local machine where the data files are stored:
+
+        md5sum * > md5sum.txt
+
+    Upload the md5sum.txt file to the same S3 folder.
+
+        aws s3 cp md5sum.txt s3://<bucket-name>/<repo-name>/v1/
+
+3. **Verify file uploads**
+    (Optional, but recommended)
+
+        aws s3 cp s3://<bucket-name>/<repo-name>/v1/md5sum.txt .
+        md5sum -c md5sum.txt
```

Comment:
I think this verification can be done on S3 and/or data download, as if you are copying back down, then you will replace the one you had generated and in case the file did not upload, you might end up pulling an old md5sum and replacing your good one with this command. I would suggest removing this or rewording in a different way. Potentially, you can pull down as a new filename `aws s3 cp s3://<bucket-name>/<repo-name>/v1/md5sum.txt md5sum-aws.txt` and then do a `diff md5sum.txt md5sum-aws.txt` to confirm they are the same. I think the check can be added above, following md5sum * > md5sum.txt, but not 100% needed since you will check the download before PR is reviewed.